tests/test_610-awkward-form.py

```diff
@@ -0,0 +1,590 @@
+import json
+import sys
+
+import numpy
+import pytest
+import skhep_testdata
+
+import uproot
+
+
+def test_awkward_strings():
+    awkward = pytest.importorskip("awkward")
```

Comment:
These can go at the top of the module file; then you'd only have to write it once.